tests/python/unit/test_binary_model.py

```diff
@@ -0,0 +1,44 @@
+import sys
+sys.path.append("..")
+print('\n'.join(sys.path))
```

Comment:
No print statements in production code :)

Reply:
@david-behar

Reply:
[60fc6be](https://github.com/dataiku/dss-plugin-generalized-linear-models/commit/60fc6beae81f3c4f49e6c1e35417964ce942b9d8)